lib/lev/routine.rb

```diff
@@ -191,6 +191,38 @@ def initialize(outputs, errors)
       end
     end
 
+    if defined?(::ActiveJob)
+      module ActiveJob
```

Comment:
since you're passing the routine in as a string, this class no longer has any real need to be an inner class.  So it would probably be better organized to make it its own class, like '/lib/lev/active_job.rb`